### PR TITLE
CONN-1758 Decision is always Permit

### PR DESCRIPTION
**Pull request by trantang**
removed the code that allow user to change the Decision
Decision Element is not removed; this created a chain of unwanted code change

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java [hunk 1]
```diff
@@ -339,15 +339,8 @@ protected Conditions createConditions(final CallbackProperties properties) {
...
-            String decision = properties.getAuthorizationDecision();
-            if (decision == null) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
-            if (!isValidAuthorizationDecision(decision)) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
+            //"Decision.  The Decision attribute of the Authorization Decision Statement must be Permit" &CONN-1758
+            String decision = AUTHZ_DECISION_PERMIT;
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java]: Unneeded local variable, just put the constant value into the method a couple lines below.

Comment (jsmith2) [reply Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java]: Also, remove the comment.  I don't think it's needed.

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java [hunk 2]
```diff
@@ -339,20 +339,10 @@ protected Conditions createConditions(final CallbackProperties properties) {
...
-            String decision = properties.getAuthorizationDecision();
-            if (decision == null) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
-            if (!isValidAuthorizationDecision(decision)) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
             final Evidence evidence = createEvidence(properties, subject);
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java:342]: Remove the isValidAuthorizationDecision and AUTHZ_DECISION_DENY method in SAMLAssertionbuilder as well.

**Review by TabassumJafri — CHANGES_REQUESTED** [review]
Please see inline comments

**Review by TabassumJafri — APPROVED** [review]
good with merge

**Review by mhpnguyen — APPROVED** [review]

**Pull request merged**